Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -975,6 +987,10 @@ def _exec_editor_multitest(self, request: _pytest.fixtures.FixtureRequest,
                 results[test_spec_name] = Result.Timeout(timed_out_result.test_spec,
                                                          results[test_spec_name].output,
                                                          self.timeout_editor_shared_test, result.editor_log)
+        finally:
+            if temp_batched_file:
+                temp_batched_file.close()
```

Comment:
why holding the handle to the file until the end? seems like an unnecessary risk, if there are exceptions or something there is the risk of not getting closed. It seems be safer to close the file after `temp_batched_file.flush()`. The deletion of the file can happen after.

Reply:
Moved closing handle to after flush()